data/sql/grid_transformation_custom.sql

```diff
@@ -147,3 +147,17 @@ INSERT INTO "usage" VALUES(
     'EPSG','1120', -- area of use: Iceland - onshore and offshore
     'EPSG','1024'  -- unknown
 );
+
+-- Japan
+
+INSERT INTO "grid_transformation" VALUES(
+    'PROJ','EPSG_7906_TO_EPSG_6695','ITRF94 to JGD2011 height',
+    NULL,NULL,
+    'EPSG','9665','Geographic3D to GravityRelatedHeight (gtx)',
+    'EPSG','6667', -- source CRS (JDG2011)
+    'EPSG','6695', -- target CRS (JDG2011 (vertical) height)
+    'EPSG','3263', -- area of use: Japan - onshore mainland
```

Comment:
You probably need to rebase your branch on top of master as the database structure has changed.
grid_transformation has no longer columns for the area of use. You need to create a record in the usage table. See above record

Reply:
I copied it from somewhere else without thinking too much.

Reply:
This time I built the `generate_proj_db` target before pushing.